client/extensions/woocommerce/app/dashboard/widgets/stats-widget/index.js

```diff
@@ -197,6 +198,9 @@ class StatsWidget extends Component {
 			<Stat
 				site={ site }
 				label={ translate( 'Conversion rate' ) }
+				labelToolTip={ translate( 'Number of orders by unique visitors', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?